pandas/util/_test_decorators.py

```diff
@@ -77,6 +77,13 @@ def safe_import(mod_name: str, min_version: str | None = None):
             return False
         else:
             raise
+    except AttributeError:
+        # TODO remove once https://github.com/scipy/scipy/issues/17827
+        # is addressed
```

Comment:
do we need a pandas issue to track this? (that scipy issue is closed as duplicate)